### PR TITLE
Add compression/META.yml

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Review by ricea — APPROVED** [review]
lgtm

File: compression/META.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+spec: https://github.com/ricea/compressstream-explainer
```
Comment (foolip) [inline compression/META.yml]: @ricea this is all I could find. If there isn't a spec yet, then all tests in this directory should be called `.tentative.`. If the design is not yet solid enough where any other vendor could plausibly make use of the tests, then you could also consider putting them in wpt_internal to not worry about these considerations yet.

Comment (ricea) [reply compression/META.yml]: https://ricea.github.io/compression/ is the temporary location of the spec. It's not quite fully-baked yet, but someone could implement it if they were prepared to deal with a bit of ambiguity.

I will rename the tests.

Comment (foolip) [reply compression/META.yml]: I'll link to https://ricea.github.io/compression/ instead.

**Pull request merged**